.github/workflows/pytest.yml

```
@@ -0,0 +1,29 @@
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.fork == true
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ "3.11" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)